contrib/auctex/packages.el

```diff
@@ -54,10 +55,36 @@
         "mhd" 'TeX-doc ;; TeX-doc is a very slow function
         )
 
+      (setq-default TeX-command-default auctex-build-command)
+
       (setq-default TeX-auto-save t)
       (setq-default TeX-parse-self t)
-      (setq-default TeX-PDF-mode t))))
+      (setq-default TeX-syntactic-comment t)
+      (setq-default TeX-PDF-mode t)
+
+      (when auctex-electric-sub-and-superscript
```

Comment:
This layer variable is not useful, users can use directly the stock `TeX-...` variable.